server/apiv2/handlers/types.go

```diff
@@ -0,0 +1,237 @@
+// Copyright 2022 TiKV Project Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package handlers
+
+import (
+	"time"
+
+	"github.com/pingcap/kvproto/pkg/metapb"
+	"github.com/pingcap/kvproto/pkg/pdpb"
+	"github.com/pingcap/kvproto/pkg/replication_modepb"
+	"github.com/tikv/pd/pkg/typeutil"
+	"github.com/tikv/pd/server/config"
+	"github.com/tikv/pd/server/core"
+)
+
+const (
+	aliveStateName        = "Alive"
+	disconnectedStateName = "Disconnected"
+	downStateName         = "Down"
+)
+
+// StoresInfo records stores' info.
+type StoresInfo struct {
```

Comment:
It seems that there is no difference between apiv1 and apiv2. How about only one declaration?

Reply:
I'm not quite sure if it will be changed later, so just copy it.